app/controllers/api/auth/episodes_controller.rb

```diff
@@ -7,13 +7,26 @@ class Api::Auth::EpisodesController < Api::BaseController
   filter_resources_by :podcast_id
   find_method :find_by_guid
 
+  after_action :process_media, only: [:create, :update]
```

Comment:
I remember there being a good reason for this, but I don't remember what. I'll try changing it and see what breaks.